src/main/java/duke/data/task/Deadline.java

```diff
@@ -0,0 +1,62 @@
+package duke.data.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Class that represents a Deadline task.
+ *
+ * @author Won Ye Ji
+ */
+public class Deadline extends Task {
+
+    protected String by;
+    private final DateTimeFormatter input = DateTimeFormatter.ofPattern("dd/MM/yy HHmm");
+    private final DateTimeFormatter output = DateTimeFormatter.ofPattern("MMM dd h:mm a");
```

Comment:
Should these be renamed to `inputFormatter` and `outputFormatter` as these are not really `input` and `output`?